src/PlaywrightSharp/NameValueEntry.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+
+namespace PlaywrightSharp
+{
+    /// <summary>
+    /// Name value entry.
+    /// </summary>
+    public class NameValueEntry : IEquatable<NameValueEntry>
```

Comment:
Is there a benefit of using this vs `KeyValuePair`?

Reply:
Let's drop it :)

Reply:
ok. we can drop it. We need to send to the driver an object with a Name/Value pair, not a Key/Value pair

Reply:
Ah, so it's a JSON serialization problem... Well, leave it, then. This might change if/when we auto-generate the channels. And we can figure it out then =) How's that?

Reply:
It will definitely change :)